apps/www/components/Sections/SectionFeed.js

```diff
@@ -90,8 +89,14 @@ const getFeedDocuments = gql`
 
 const mapNodes = (node) => node.entity
 
-const SectionFeed = ({ t, formats, variablesAsString }) => {
-  if (!variablesAsString && !(formats && formats.length)) {
+const SectionFeed = ({
+  t,
+  formats,
+  variables = {
+    filter: { formats, feed: true },
+  },
+}) => {
+  if (!variables && !(formats && formats.length)) {
```

Comment:
This check no longer works because `variables` with the default value will never be falsy. Easies way: keep it like before where the default value is constructed after this check.

Reply:
done